<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Surjits-Acer-TravelMate-TMP449-G2-M" ignored="0" name="Default test" tests="2" failures="1" timestamp="2021-01-15T18:52:13 BDT" time="21.348" errors="0">
  <testcase name="dmanager" time="20.507" classname="com.bitm.SeleniumOnlineProject3.Test.AssignLeaveTest"/>
  <testcase name="assignLeaveTest" time="0.0" classname="com.bitm.SeleniumOnlineProject3.Test.AssignLeaveTest">
    <failure type="org.testng.TestNGException" message="
Method public void com.bitm.SeleniumOnlineProject3.Test.AssignLeaveTest.assignLeaveTest(java.util.List) requires a @DataProvider named : assignleavedata in class com.bitm.SeleniumOnlineProject3.DataProvider.AssignLeaveDataProvider">
      <![CDATA[org.testng.TestNGException: 
Method public void com.bitm.SeleniumOnlineProject3.Test.AssignLeaveTest.assignLeaveTest(java.util.List) requires a @DataProvider named : assignleavedata in class com.bitm.SeleniumOnlineProject3.DataProvider.AssignLeaveDataProvider
at org.testng.internal.Parameters.findDataProvider(Parameters.java:559)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- assignLeaveTest -->
</testsuite> <!-- Default test -->
